lib/super_good/solidus_taxjar/tax_calculator.rb

```diff
@@ -109,8 +109,14 @@ def no_tax
         )
       end
 
+      # Tax adjustments require an associated Tax Rate. This tax rate is not
+      # used in the extension's calculations as the rates from the TaxJar API
+      # are used, so a placeholder rate and calculator is created.
       def tax_rate
-        ::Spree::TaxRate.find_by(name: "Sales Tax")
+        ::Spree::TaxRate.find_or_create_by!(name: "TaxJar Placeholder Rate") do |tax_rate|
+          tax_rate.calculator = ::Spree::Calculator.new
+          tax_rate.amount = 0
```

Comment:
What's the purpose of this line? I noticed it wasn't tested so I was wondering if it's important it should maybe have a test to go along with the change, if it's not important do we need it at all?

Reply:
The amount is not used at all as the rate is just used as a placeholder and the real tax rate is calculated by the TaxJar API

I was basically trying to setup the properties of the rate in the most generic way possible to signify it is a placeholder rate and doesn't have a real impact, and setting the amount to `0` seemed like a good value to communicate that!